src/ankita/FindMaximumWord.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * 1) print name having maximum length and print its length also.
+Add Adv : if you can scanner class.
+String[] name = {"Harsh","maulik","Technocredits","Krishna"};
+output : Technocredits -> 13
+
+	String getMaxLengthName(String[] arr){
+	{
+		return name;
+	}
+
+2) display method to print all even charcter of the name having maximum length.
+   void display(String name){
+   
+   }
+ */
+package ankita;
+
+public class FindMaximumWord {
+	String findMaxWord(String[] arr) {
+		int max =0;
+		String maxWord=arr[0];
+		for(int index=0;index<arr.length;index++) {
+			String name = arr[index];
+			int cnt=0;
+			for(int i =0;i<name.length();i++) {
```

Comment:
why do we need loop ????
can we directly say cnt= name.length() ???? even we dont need cnt.
dont create unnecessary vairables

Reply:
Corrected.